pathways/simulation.py

```diff
@@ -204,6 +208,7 @@ def simulation(
     simulation_results = types.SimpleNamespace(
         missing=missing,
         false_neg=false_neg,
+        missed_tolerance=missed_tolerance,
```

Comment:
Can this be something like `missed_in_tolerance` to make it clear that it is "slipped within tolerance"? Longer, but perhaps worth it.